docs/resource-policy/policy/balloons.md

```diff
@@ -106,6 +106,15 @@ Balloons policy parameters:
   - `namespaces` is a list of namespaces (wildcards allowed) whose
     pods should be assigned to this balloon type, unless overridden by
     pod annotations.
+  - `matchContainers` is a list of container match expressions. These
+    expressions are evaluated for all containers which have not been
+    assigned otherwise to other balloons. If an expression matches,
+    IOW it evaluates to true, the container gets assigned to this
+    balloon type. Container mach expressions have the same syntax and
+    semantics as the scope and match expressions in container affinity
+    annotations for the topology-aware policy.
+    See the [affinity documentation](./topology-aware.md#affinity-semantics)
+    for a detailed description of expressions.
```

Comment:
Essentially it is similar to `nodeAffinity`. Have you thought about the having a support for the `AntiAffinity` as well?

Reply:
No, not here in the balloons policy. In topology-aware, where the whole concept was introduced first, we do have anti-affinities supported. There it is theoretically possible to come up with not too weird semantics for the concept of a 'distance between two topology zones', so anti-affinity is not too difficult conceptually (although the current implementation might not be too good for 'diluted affinities and anti-affinities'). For balloons I think anti-affinity would be conceptually more problematic.

Once thing not quite unlike anti-affinity that could make sense for balloons is an `exceptContainers` which would have match expression for containers that you do not want to put into the balloon even if other conditions would otherwise dictate the contrary. But I would not add it now, since it is not needed at the moment.